### ISSUE TITLE
[week5] TODO

### ISSUE THREAD

**Issue by kimcando** [kimcando/BoostcampAITech3-PaperReading-Embedding#21]
* word2vec의  cbow, skip-gram 구현 방법 및 구현 차이
* genism 모듈을 이용한 wod2vec 사용 및 간단한 소개?
* doc2vec 구현 코드 참고 : https://github.com/inejc/paragraph-vectors
* negative sampling & subsampling 차이
* hierarchical softmax 좀더 설명 + 구현이 어떻게 되는지?
* Noise Contrastive Estimation(NCE) 좀더 설명

**Comment by HanNayeoniee** [comment]
- cbow 구현 [CBOW 임베딩 학습하기](https://didu-story.tistory.com/102?category=952805)
- skip-gram 구현 [udacity-skip-gram-solution](https://github.com/udacity/deep-learning-v2-pytorch/blob/master/word2vec-embeddings/Skip_Grams_Solution.ipynb), [skip-gram 예제](https://direction-f.tistory.com/29)

**Comment by kimcando** [comment]
* hierarchical softmax 설명

**Comment by angieKang** [comment]
negative sampling & subsampling

**Comment by TB2715** [comment]
genism 모듈을 이용한 wod2vec 사용 및 간단한 소개
